configs/rpi4-arcade-min.buildercfg

```diff
@@ -1,3 +1,4 @@
 raspberrypi4_64_partconfig
+disable_features_partconfig
```

Comment:
Note that this is the best way to do it right now but maybe there could be a way to add stacking so if one has something not set and a later one adds it, that would take precedence. This could go along with a include system if we wanted to go down that path.